lib/v2/tableau/viz-of-the-day.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+// const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    // Your logic here
+    const rootUrl = 'https://public.tableau.com/api/gallery?page=0&count=20&galleryType=viz-of-the-day';
+    const { data: response } = await got(rootUrl);
+
+    const items = response.items.map((item) => ({
+        title: item.title,
+        link: item.sourceUrl,
+        pubDate: parseDate(item.galleryItemPublicationDate),
+        author: item.authorName,
+        description: item.description,
```

Comment:
Consider adding `screenshot` to the description as well.

Reply:
tks, I figured out how to add new html into description, and it is done